make all no longer exists with success

I pulled the latest changes and now get this:

```
/tmp/ccs6ZpC7.s: Assembler messages:
/tmp/ccs6ZpC7.s:96: Error: unrecognized keyword/register name `movts fstatus,r10'
/tmp/ccs6ZpC7.s:104: Error: unrecognized keyword/register name `movts resetcore,r10'
<builtin>: recipe for target 'setallregs' failed
make: *** [setallregs] Error 1
```

This is using the 2015.1 version of the eSDK (installed similarly to the script in [this Docker container](https://github.com/futurecore/parallella-devenv/blob/master/Dockerfile)).

Am I using the wrong version of the SDK?

--- a/setallregs.c
+++ b/setallregs.c
@@ -7,7 +7,7 @@ int main()
 //  __asm__ ("add r10, r10, 1");
 //  __asm__ ("movts pc, r10");
 //  __asm__ ("add r10, r10, 1");
-  __asm__ ("movts DEBUG, r10");
+  __asm__ ("movts debugstatus, r10");
   __asm__ ("add r10, r10, 1");
   __asm__ ("movts lc, r10");
   __asm__ ("add r10, r10, 1");

